utils/vara-ui/src/components/input/input.module.scss

```diff
@@ -127,34 +158,97 @@
   bottom: 0;
   left: 0;
   right: 0;
+  z-index: -1;
 
-  border: 1px solid #0000003b;
   border-radius: 4px;
+  background-color: #ffffff;
```

Comment:
We can't put white background color as we can't guarantee that every application's background will be white too. It should be transparent.